issuetracker/src/main/java/org/openjdk/skara/issuetracker/jira/JiraIssue.java

```diff
@@ -39,7 +39,6 @@ public class JiraIssue implements IssueTrackerIssue {
     private final JiraProject jiraProject;
     private final RestRequest request;
     private final JSONValue json;
-    private final boolean needSecurity;
```

Comment:
I think we should keep this boolean and use `jiraProject.jiraHost().visibilityRole().isPresent()` to initialize it. Preventing editing of title and body is still relevant.